src/pybind/mgr/dashboard/frontend/src/app/shared/services/summary.service.ts

```diff
@@ -14,7 +14,8 @@ import { ServicesModule } from './services.module';
 export class SummaryService {
   // Observable sources
   private summaryDataSource = new BehaviorSubject(null);
-
+  // True after the summary data has been loaded the first time.
+  loaded = false;
```

Comment:
Please make sure with unit tests that their won't be any regression regarding this bug.